plugins/modules/ipa_host.py

```diff
@@ -74,10 +74,16 @@
     type: list
     elements: str
   state:
-    description: State to ensure.
+    description:
+    - State to ensure.
     default: present
     choices: ["absent", "disabled", "enabled", "present"]
     type: str
+  force_creation:
+    description:
+    - Create host if O(state=disabled) or O(state=enabled) but not present.
+    default: false
```

Comment:
Shouldn't the default be `false` for backwards compatibility?

Reply:
you meant `true`, right? :)

Reply:
LOL, yes :)